### DISCUSSION TITLE
How can I prevent Modal close when Menu is closed

### DISCUSSION THREAD

**Discussion by phuriwatcmk** [mantinedev/mantine#1407] [Q&A] [answered]
I make some a navbar with Menu and the menu have Setting(Item Menu) with Modal When I clicked the Setting(Item Menu) Modal They are both disappear  

I have a clip that show my problem I'm a newbie in react. Thanks for your kinds

**Comment by rtivital** [comment, marked as answer]
- Do not render Modal within Menu.Item
- When Modal is opened close menu
- You control modal state with `opened` prop on your side, so it's up to you to decide when modal will be closed

**Reply by phuriwatcmk** [reply]
It's work, Thanks for your response.
